packages/icons-angular/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@carbon/icons-angular",
+  "description": "Carbon icons in Angular",
+  "version": "0.0.1-alpha.14",
+  "license": "Apache-2.0",
+  "author": "IBM",
+  "main": "lib/index.js",
+  "repository": "https://github.com/IBM/carbon-elements/tree/master/packages/icons-angular",
+  "bugs": "https://github.com/IBM/carbon-elements/issues",
+  "files": [
+    "lib"
```

Comment:
Technically _no_ since the common usecase includes some form of bundler running the angular compiler (or at the very least a TS compiler) ... That said, it is common to export a UMD as well, so we may as well add that in (full bundle + individual icon UMDs)